packages/google_maps_flutter/google_maps_flutter_web/lib/src/convert.dart

```diff
@@ -81,11 +81,18 @@ gmaps.MapOptions _rawOptionsToGmapsOptions(Map<String, dynamic> rawOptions) {
     options.styles = rawOptions['styles'];
   }
 
-  if (rawOptions['scrollGesturesEnabled'] == false ||
+  if (rawOptions['gestureHandling'] != null) {
+    options.gestureHandling = rawOptions['gestureHandling'];
+  } else if (rawOptions['scrollGesturesEnabled'] == false ||
       rawOptions['zoomGesturesEnabled'] == false) {
```

Comment:
This should also take into account `rawOptions['rotateGesturesEnabled']` (they may be available in the web [soon?](https://developers.google.com/maps/documentation/javascript/webgl?hl=en)).

Reply:
Yes, it seems that with some configuration in the GCP console, the map becomes "WebGL", and in that case, rotation and tilt become available on the web (unless disabled from the console, which I guess keeps the WebGL rendering but disables rotation and tilt, to keep things "backwards compatible") ((I'm not in the Maps team, so I don't have any info on why they decided to make this feature opt-in like they did, sorry!))